docs/CLI.md

```diff
@@ -49,6 +49,9 @@ selenium-standalone start --singleDriverStart=chrome
 selenium-standalone install --config=/path/to/config.json
 selenium-standalone start --config=./config/seleniumConfig.js
 
+# killing process before starting
+selenium-standalone start --processKiller=true
```

Comment:
A user would ask why they would need to set this given it feels like the expected behavior is that the selenium server or driver shuts down completely when the process is triggered to get shutdown.